packages/mdl-drawer/temporary/index.js

```diff
@@ -0,0 +1,138 @@
+/**
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import MDLComponent from 'mdl-base';
+import MDLTemporaryDrawerFoundation from './foundation';
+
+export default class MDLTemporaryDrawer extends MDLComponent {
+  static buildDom() {
+    const {ROOT: CSS_ROOT} = MDLTemporaryDrawerFoundation.cssClasses;
+
+    const root = document.createElement('aside');
+    root.classList.add(CSS_ROOT);
+    root.innerHTML = `<nav class="${CSS_ROOT}__drawer"></nav>`;
+
+    return root;
+  }
+
+  static attachTo(root) {
+    return new MDLTemporaryDrawer(root);
+  }
+
+  /* Remap touch events to pointer events, if the browser supports them. */
+  static remapEvent(eventName) {
```

Comment:
Where do you see this file living? Inside `mdl-drawer`, or inside its own module? For the latter, I'd say it's probably best to file an issue and refactor that out later.